scene/gui/label.cpp

```diff
@@ -487,6 +487,9 @@ void Label::_notification(int p_what) {
 					}
 				}
 			}
+			if (i >= lines_rid.size()) {
+				break;
+			}
```

Comment:
What's the purpose of this part of the PR?

Reply:
I can separate it. It's a different crash. Not sure how to describe it.

It's the same as checking if i is inside of lines_rid but without the warning message.

Reply:
Probably better to separate it then. Also it would be nice to have steps to reproduce the crash without the fix. You can describe them directly in a PR if you want.

Reply:
When is it crashing? `i` is already limited by `int last_line = MIN(lines_rid.size(), lines_visible + lines_skipped)`, this condition should never be true.